File: reservation-manager/add_guess.cpp
#include <iostream>
#include <fstream>
#include "json.hpp"

using json = nlohmann::json;

int main() {
    json data;

    std::ifstream in("people.json");
    if (in.is_open()) {
        in >> data;
    }

    std::string name;
    int age;

    std::cout << "Name: ";
    std::cin >> name;

    std::cout << "Age: ";
    std::cin >> age;

    data["people"].push_back({
        {"name", name},
        {"age", age}
    });

    std::ofstream out("people.json");
    out << data.dump(4);
}

File: reservation-manager/add_guess.py
import json
import os

def adding_guess():
    print("\n")
    if not os.path.exists("guess.json") or os.path.getsize("guess.json") == 0:
         data = []
    else:
       with open("guess.json", "r") as file:
        data = json.load(file)
    name_of_the_guess = input("Enter Frist & Last Name: ")
    age_of_the_guess = input("age: ")
    
    data.append({"name": name_of_the_guess, "age": age_of_the_guess})

    with open("guess.json", "w") as file:
     json.dump(data, file, indent=4)
     
    print("Guess added")